mtgjson5/cache_builder.py

```diff
@@ -0,0 +1,473 @@
+"""
+Global cache for MTGJSON provider data and pre-computed aggregations.
+"""
+
+import json
+import ijson
```

Comment:
Do we need both json and ijson? If so, you'll need to add ijson to the requirements